curvefs/src/client/fuse_client.cpp

```diff
@@ -347,7 +334,7 @@ CURVEFS_ERROR FuseClient::FuseOpSetAttr(
     if (to_set & FUSE_SET_ATTR_MTIME) {
         inode.set_mtime(attr->st_mtime);
     }
-    uint64_t nowTime = TimeUtility::GetTimeofDayMs();
+    uint64_t nowTime = TimeUtility::GetTimeofDaySec();
```

Comment:
why change this percision ?

Reply:
> why change this percision ?

The time unit of atime/ctime/mtime in inode need to fix later.